third_party/blink/renderer/modules/mediasource/source_buffer.h

```diff
@@ -113,6 +113,10 @@ class SourceBuffer final : public EventTargetWithInlineData,
   AudioTrackList& audioTracks();
   VideoTrackList& videoTracks();
 
+  // Cobalt-specific method that returns the highest presentation
+  // timestamp written to the SbPlayer.
+  double writeHead(ExceptionState& exception_state) const;
```

Comment:
Should this be gated by `USE_STARBOARD_MEDIA`?

Reply:
Yep, I'll add that and remove the RuntimeEnabledFeature